src/components/reddit/RedditReturn.jsx

```diff
@@ -1,17 +1,49 @@
 import { useSelector } from "react-redux";
 import { selectRedditReturn } from "./RedditSlice";
 import Reddit from "./Reddit";
+import ChildObject from "./ChildObject";
 
 const RedditReturn = () => {
-    const redditReturn = useSelector(selectRedditReturn);
+    let returnedRedditArray = [];
+    const returnedArray = useSelector(selectRedditReturn);
+    const redditReturn = () => {
+        if(!returnedArray) {
+            return ;
+        }
+        for(let i = 2; i < returnedArray.length; i++) {
```

Comment:
you don't need for loops in this code. you should do things using native javascript functions. Just get used to doing it that way